### DISCUSSION TITLE
Troubleshooting Zwave Mesh Traffic Issues

### DISCUSSION THREAD

**Discussion by alexruffell** [zwave-js/zwave-js-ui#4040] [Ideas]
With the addition of controller and node statistics, one can now look for nodes that cause trouble on the mesh network by being too chatty or by dropping too many messages. However, it is a very time consuming process as you have to expand each node to look at its statistics.

Would it be possible to have a view in table format where the statistics (customizable by the user would be a plus) are shown for all devices in a mesh network? This would make it so much easier to find those pesky devices!

Some additional metrics that could be calculated could be percentage of total as a mere count may not be reflective of its contribution to the traffic. Automatically highlighting devices that are beyond some thresholds might help too. Thoughts?

**Comment by robertsLando** [comment]
Not sure about the best way to represent this on UI.

@AlCalzone thoughts?

**Reply by AlCalzone** [reply]
Would be cool I think. For a first iteration, a simple table with a row per node and columns for the individual statistics would probably suffice.
